symbols/dpst_switch_horz.ts

```diff
@@ -0,0 +1,22 @@
+import { defineSymbol } from "drawing/defineSymbol"
+import svgJson from "assets/generated/dpst_switch.json"
+import { Primitive } from "drawing/types"
+
+const { paths, texts, bounds, refblocks, circles } = svgJson
+
+export default defineSymbol({
+  primitives: [
+    ...Object.values(paths),
+    ...Object.values(circles),
+    { ...texts.top1, anchor: "middle_bottom", x: 0 },
+    { ...texts.bottom1, anchor: "middle_top", x: 0 },
+  ] as Primitive[],
+  ports: [
+    { ...refblocks.left1, labels: ["1"] }, // TODO add more "standard" labels
```

Comment:
Can we add `left1`, `right1` etc. for these? Really great work, I love it so much we can merge and fix this later/separately

```suggestion
    { ...refblocks.left1, labels: ["1", "left1"] }, // TODO add more "standard" labels
```